apertium-cat.cat.dix

```diff
@@ -5121,6 +5121,13 @@
 
 </pardef>
 
+<pardef n="semide/u__n">
+  <e>       <p><l>eu</l>         <r>u<s n="n"/><s n="m"/><s n="sg"/></r></p></e>
+  <e>       <p><l>essa</l>      <r>u<s n="n"/><s n="f"/><s n="sg"/></r></p></e>
+  <e>       <p><l>us</l>        <r>u<s n="n"/><s n="m"/><s n="pl"/></r></p></e>
+  <e>       <p><l>esses</l>     <r>u<s n="n"/><s n="f"/><s n="pl"/></r></p></e>
+</pardef>
+
```

Comment:
It's OK to add the feminine form. But we must keep the form with diacritic (semidéu). The words with diacritics changed in the orthography reform will be dealt with in a post-edition module.